themes/aura/assets/product-slider.css

```diff
@@ -1,3 +1,16 @@
+@media (min-width: 768px){
```

Comment:
```suggestion
.product-slider.yc-slider__desktop{
  display:none;
}
@media (min-width: 768px){
  .product-slider.yc-slider__mobile{
    display:none;
  }
  .product-slider.yc-slider__desktop{
    display:block;
  }
}
```

Reply:
this code is generated and do not have any action to do with it